packages/material-ui/src/styles/colorManipulator.js

```diff
@@ -198,15 +198,45 @@ export function emphasize(color, coefficient = 0.15) {
   return getLuminance(color) > 0.5 ? darken(color, coefficient) : lighten(color, coefficient);
 }
 
+let warnedOnce = false;
+
 /**
  * Set the absolute transparency of a color.
  * Any existing alpha values are overwritten.
  *
  * @param {string} color - CSS color, i.e. one of: #nnn, #nnnnnn, rgb(), rgba(), hsl(), hsla()
  * @param {number} value - value to set the alpha channel to in the range 0 -1
  * @returns {string} A CSS color string. Hex input values are returned as rgb
+ *
+ * @deprecated
+ * Use `import { alpha } from '@material-ui/core/styles'` instead.
  */
 export function fade(color, value) {
+  if (process.env.NODE_ENV !== 'production') {
+    if (!warnedOnce) {
+      warnedOnce = true;
+      console.error(
+        [
+          'Material-UI: the fade color utility was renamed to alpha to better describe the functinality of the utility.',
```

Comment:
```suggestion
          'Material-UI: the `fade` color utility was renamed to alpha to better describe its functionality.',
```
Not 100% sure about using backticks in console output, since they won't be transformed, but I personally think it's clearer.

Reply:
I haven’t seen so far usages of backticks in the console errors, but I agree that it emphasizes the name of the method 🤷‍♀️ Also we should probably used them on the alpha name as well I guess?